plugins/plugin-svn/che-plugin-svn-ext-shared/src/main/java/org/eclipse/che/plugin/svn/shared/UpdateRequest.java

```diff
@@ -119,4 +120,22 @@
      */
     UpdateRequest withRevision(@NotNull final String revision);
 
+    /** @return user name for authentication */
+    String getUserName();
+
+    /** Set user name for authentication. */
+    void setUserName(@Nullable final String username);
```

Comment:
Parameter name is `username` but method name is `setUserName`. Please name in the same way. I prefer to use `username`